src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,33 +1,121 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
     # TO-DO
-    
+    i = 0
+    j = 0
+    k = 0
+    while j < len(arrA) and k < len(arrB):
+        if arrA[j] <= arrB[k]:
+            merged_arr[i] = arrA[j]
+            j += 1
+        else:
+            merged_arr[i] = arrB[k]
+            k += 1
+        i += 1
+    while j < len(arrA):
+        merged_arr[i] = arrA[j]
+        j += 1
+        i += 1
+    while k < len(arrB):
+        merged_arr[i] = arrB[k]
+        k += 1
+        i += 1
     return merged_arr
```

Comment:
Excellent! See if you can refactor merged_arr down to around 15 lines instead of 25ish

Reply:
Whoops! I meant the merge() function